packages/firestore/CHANGELOG.md

```diff
@@ -6,6 +6,8 @@
   if you continue to use `experimentalTabSynchronization`.
 - [feature] You can now query across all collections in your database with a
   given collection ID using the `FirebaseFirestore.collectionGroup()` method.
+- [changed] Firestore now recovers more quickly from long periods without
+  network access (#1809).
```

Comment:
I see you modeled this off Android, but:
1) This isn't really specific to "long periods without network access"
2) We usually don't include the PR number in changelog entries.  If there is a customer-reported issue that the PR is fixing (and that issue contains useful context), then sometimes we *do* reference it (e.g. "Fixes a potential crash when reading from persistence (#<insert issue ID with all the gory details)").  But all else being equal, ideally the changelog entry will be sufficient to describe the change on its own.

So I propose something like: Firestore now recovers more quickly after network connectivity changes (airplane mode, wifi / cellular availability, etc.).